medic/medic-run.js

```diff
@@ -232,12 +233,18 @@ function windowsSpecificPreparation(argv) {
             /^\s*\$appActivator .*$/gim,
             "$&\n" +
             "    powershell " + path.join(process.cwd(), destScriptPath) + " $$ID\n" +
+            "    powershell " + path.join(process.cwd(), logScriptPath) + "\n" +
             "    $Ole32 = Add-Type -MemberDefinition '[DllImport(\"Ole32.dll\")]public static extern int CoAllowSetForegroundWindow(IntPtr pUnk, IntPtr lpvReserved);' -Name 'Ole32' -Namespace 'Win32' -PassThru\n" +
             "    $Ole32::CoAllowSetForegroundWindow([System.Runtime.InteropServices.Marshal]::GetIUnknownForObject($appActivator), [System.IntPtr]::Zero)",
             appUtilsPath
         );
     }
 
+    // write current time to a file to use it when gathering logs
+    var now = new Date();
+    var iso = now.toISOString();
+    fs.writeFileSync('startTime.txt', iso, util.DEFAULT_ENCODING);
```

Comment:
This file is not necessary. It serves only to pass arguments from `medic-run` to `medic-log`. Instead, please pass a start time to medic-log on the command line from `cordova.conf`. You can do this by setting a `Property` using `SetPropertyFromCommand`, and then passing it to the logging step.